.github/workflows/rust.yml

```diff
@@ -11,16 +11,54 @@ env:
 
 jobs:
   build:
-
+    name: Build
     runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v3
+    - uses: dtolnay/rust-toolchain@master
+      with:
+        toolchain: 1.65.0
```

Comment:
```suggestion
    - uses: dtolnay/rust-toolchain@1.65.0
```

Reply:
Ah, yes we could do that.

I did not because I guessed that you might want to move to matrix builds at some point, ensuring that you're building with an old version as MSRV _and_ stable. But I can convert it of course, if you want!

Reply:
I was assuming that `...@{{ matrix.toolchain.rust }}` would work. But let’s cross that bridge when we get there … libp2p progresses quite quickly and doesn’t care much about MSRV, so I didn’t have imminent plans for that either.

Reply:
Yes, that would work. Its `dtolnay/rust-toolchain@${{ matrix.rust }}`, but as you said, lets use 1.65.0 only for now.

Fixed it.